app/src/main/java/ch/epfl/sweng/SDP/home/HomeActivity.java

```diff
@@ -434,4 +434,15 @@ private void setLeague() {
         leagueText.setTextColor(getResources().getColor(getLeagueColorId(league)));
         ((ImageView) findViewById(R.id.leagueImage)).setImageResource(getLeagueImageId(league));
     }
+
+    @Override
+    public void onResume() {
```

Comment:
If the home activity is launched by an exit button, we don’t need this, and @LudoHoff will do this change in all the activities which go back to the home. Maybe we just have to annotate this method as « to be removed » once the mentioned changes have been done

Reply:
Yes I see, so we would need to add a button to leave the shop and remove back button functionality.